DATASETS.md

```diff
@@ -0,0 +1,97 @@
+
+There are four steps to adding your dataset:
+1) Add the dataset's resources.
+2) Add the JSON file describing the dataset's publication (note: this is different than for publications!)
+3) Add the JSON file describing the dataset.
+4) Validate your dataset.
+
+Ensure that all files you add are named appropriately. Datasets follow the `<name-of-dataset>.json` convention (i.e. kebab-case), while publications follow the convention spelled out above.
+
+### Adding the Datasets Resources
+
+Dataset files are hosted at [https://linqs-data.soe.ucsc.edu/public/datasets/](https://linqs-data.soe.ucsc.edu/public/datasets/).
+To add your files simply ssh into a server then navigate to the datasets folder in the LINQS public data directory, create a directory called `<name-of-dataset>`, and add a **compressed** version of your dataset with the same name as the directory you just created.
+
+### Adding the Publication JSON
+
+#### Your Dataset is Published in a LINQS Publication
+
+If your dataset is published in a LINQS publication, then you should add and validate the paper's metadata in the style of a publication.
+Once you have added the publication, the dataset needs access to that data, so you should create a soft link to the publication's data in the `\datasets\pubs` folder.
+This can be done using the following bash command `ln -s <path-to-destination>.json <source>.json` in `\datasets\pubs`.
+The name of this file will be used as a key in the dataset metadata file to provide additional information.
+
+#### Your Dataset is **NOT** Published in a LINQS Publication
+
+If your dataset is **not** published in a LINQS publication, then create a valid publication JSON, but rather than putting it in `_data/pubs/` put it in `_data/datasets/pubs/`.
+This will need to pass the same validation test as regular publications, so be sure to follow the same formatting.
+
+### Adding the Dataset JSON
+
+Now we can add the information about the dataset itself. First create a file called `<name-of-dataset>.json` in `_data/datasets/metadata/`.
+The easiest way to create the content of this file, is by copying one of the existing dataset metadata files.
+
+The following fields are required:
+* `title`
+* `description`
+* `citation`
+* `link`
+* `references`
+
+The following fields are optional, but should be included unless there is good reason for why you can't:
+* `link`: `md5`, `size`
+
+#### `title`
+
+This is the title of your dataset.
+It will be displayed as is.
+
+#### `description`
+
+This description should give a user information about what the dataset is used for, as well as what it contains, and how it was collected.
+
+#### `citation`
+
+The citation specifies a file in `_data/datasets/pubs/`. It should be the same as the name of the publication file you want rendered as bibtex.
```

Comment:
I believe this link was changed, is this true?
Verify all the links in the readme.